### PR TITLE
feat(component): TextField component supper number type

### PR DESCRIPTION
**Pull request by gavin1ee**
feat(component): TextField component supper number type

### CODE REVIEW

File: src/Component/BlazorComponent/Components/TextField/Content/Control/InputSlot/DefaultSlot/NumberIconSlot/BTextFieldNumberIconSlot.razor [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿@namespace BlazorComponent
+@typeparam TValue
+@typeparam TInput
+@inherits ComponentPartBase<TInput>
+
+@if (Type == "number" && Props.ShowControl)
+{
+    var upButtonchildContent = RenderPart(typeof(BInputIcon<,>), "append-icon-number-up", "mdi-chevron-up", HandleOnNumberUpClickAsync, arg0Name: "Type", arg1Name: "Icon", arg2Name: "OnClick");
```
Comment (capdiem) [inline src/Component/BlazorComponent/Components/TextField/Content/Control/InputSlot/DefaultSlot/NumberIconSlot/BTextFieldNumberIconSlot.razor:8]: child -> Child

File: src/Component/BlazorComponent/Components/TextField/Content/Control/InputSlot/DefaultSlot/NumberIconSlot/BTextFieldNumberIconSlot.razor [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    var downButtonChildContent = RenderPart(typeof(BInputIcon<,>), "append-icon-number-down", "mdi-chevron-down", HandleOnNumberDownClickAsync, arg0Name: "Type", arg1Name: "Icon", arg2Name: "OnClick");
+    <div class="@CssProvider.GetClass("number-wrap")">
+        @RenderPart(typeof(BInputSlot<,>),"append","inner",downButtonChildContent,arg0Name:"Type","Location","ChildContent")
```
Comment (capdiem) [inline src/Component/BlazorComponent/Components/TextField/Content/Control/InputSlot/DefaultSlot/NumberIconSlot/BTextFieldNumberIconSlot.razor:11]: add a whitespace after comma.

File: src/Component/BlazorComponent/Components/TextField/TextFieldNumberProperty.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+{
+    public class TextFieldNumberProperty
+    {
+        public decimal? Min { get; set; }
+
+        public decimal? Max { get; set; }
+
+        public decimal Step { get; set; } = 1;
+
+        public bool ShowControl { get; set; } = true;
```
Comment (capdiem) [inline src/Component/BlazorComponent/Components/TextField/TextFieldNumberProperty.cs:17]: 改成HideControl如何？ @doddgu

**Review by capdiem — CHANGES_REQUESTED** [review]

**Review by capdiem — APPROVED** [review]
LGTM. @doddgu

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**